packages/flutter/test/widgets/baseline_test.dart

```diff
@@ -93,6 +93,28 @@ void main() {
     await tester.pump();
     expect(calls, 2);
   });
+
+  testWidgets('Baseline in LayoutBuilder is passed to child', (WidgetTester tester) async {
+    int calls = 0;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Baseline(
+            baseline: 100.0,
+            baselineType: TextBaseline.alphabetic,
+            child: LayoutBuilder(
+              builder: (BuildContext context, BoxConstraints constraints) {
+                return BaselineDetector(() {
```

Comment:
Can you modify the test to just call the getDistanceToBaseline on renderobject of LayoutBuilder and see if it returns the right value when it has child?

Reply:
If I guessed right, I won't be able to call getDistanceToBaseline in the test after getting the LayoutBuilder RenderBox since it's only allowed to call when performing layout. If I guessed it wrong, can you provide any widget name which has such a test, it would be a great help.

Reply:
I have edited the test to check that getDistanceToActualBaseline is actually caching the baseline for computation as we have done for other widgets like Chip and ListTile.

Reply:
This is not a good test for this change, can we try to build a real child and verify its position does take the child.getDistanceToActualBaseline into account?

Reply:
I have updated the test. Let me know if it's not good. I'll try to improve.